src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt

```diff
@@ -40,31 +40,47 @@ class AlertsSynchronisationService(
     if (nomisAlert.audit.auditModuleName == "DPS_SYNCHRONISATION") {
       telemetryClient.trackEvent("alert-synchronisation-created-skipped", telemetry)
     } else {
-      val mapping = mappingApiService.getOrNullByNomisId(event.bookingId, event.alertSeq)
-      if (mapping != null) {
-        telemetryClient.trackEvent(
-          "alert-synchronisation-created-ignored",
-          telemetry + ("dpsAlertId" to mapping.dpsAlertId),
-        )
+      if (nomisAlert.audit.auditAdditionalInfo == "OMKCOPY.COPY_BOOKING_DATA") {
```

Comment:
Would this be easier to understand with and `else if`? I'm getting a little lost

Reply:
Maybe I will extract into methods